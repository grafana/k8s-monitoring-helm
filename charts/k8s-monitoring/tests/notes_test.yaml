# yamllint disable rule:document-start rule:line-length rule:trailing-spaces
suite: Feature - Notes
templates:
  - NOTES.txt
tests:
  - it: print the warning about ecosystem translation if the destinations don't match
    set:
      cluster:
        name: ci-test-cluster
      destinations:
        - name: otlp
          type: otlp
          url: http://otlp.otlp.svc:4317
          logs: {enabled: true}
      clusterEvents:
        enabled: true
      alloy-singleton:
        enabled: true
    asserts:
      - matchSnapshotRaw: {}
