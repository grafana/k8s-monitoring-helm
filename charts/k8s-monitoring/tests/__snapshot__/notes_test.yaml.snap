print the warning about ecosystem translation if the destinations don't match:
  1: |
    |
      Grafana k8s-monitoring Helm chart deployed!

      This chart will install the following components:
      * Grafana Alloy "alloy-singleton" (deployment, 1 replica)

      It will:
      * Gather Kubernetes Cluster events and send data to otlp*.



      * One or more features will have their data translated to a different storage ecosystem. This may result in an imperfect
        translation of labels and attributes. For more information, consult:
          https://github.com/grafana/k8s-monitoring-helm/tree/main/charts/k8s-monitoring/docs/destinations#ecosystem-translation
